recipes/sfepy/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set rev = "d479f7a9" %}
+{% set name = "sfepy" %}
+{% set org = "sfepy" %}
+{% set version = "d479f7a9" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/{{ org }}/{{ name }}.git
+  git_rev: {{ rev }}
+
+build:
+  number: 0
+  script: CFLAGS='-I${PREFIX}/include/python${CONDA_PY}m' python setup.py install --single-version-externally-managed --record record.txt
+  skip: true  # [win]
+
+requirements:
+  build:
+    - python
+    - anaconda
+    - cython
+    - sympy
+    - pytables
+    - mpi4py
+    - scikit-umfpack
+
+  run:
+    - python
+    - anaconda
+    - pyparsing
+    - sympy
+    - cython
+    - pytables
+    - mpi4py
+    - scikit-umfpack
+
+about:
+  home: http://{{ name }}.org
+  license: BSD-3-Clause
+  license_file: LICENSE
+  summary: "Simple finite elements in Python"
+  description: >-
+    SfePy is a software for solving systems of coupled partial
+    differential equations (PDEs) by the finite element method in 1D,
+    2D and 3D. It can be viewed both as black-box PDE solver, and as a
+    Python package which can be used for building custom
+    applications. The word simple means that complex FEM problems can
+    be coded very easily and rapidly.
+  doc_url: http://sfepy.org
+  dev_url: https://github.com/sfepy/sfepy
+
+extra:
+  recipe-maintainers:
+    - rc
```

Comment:
@rc Are you ok with being listed as a co-maintainer on this conda-forge package?